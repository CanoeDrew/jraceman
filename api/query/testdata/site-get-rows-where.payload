[
  {
    "Name": "id",
    "Op": "eq",
    "Value": "S1"
  }
]
