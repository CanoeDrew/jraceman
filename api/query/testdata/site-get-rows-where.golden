{
  "Columns": [
    {
      "Name": "id",
      "Type": "string"
    },
    {
      "Name": "name",
      "Type": "string"
    },
    {
      "Name": "street",
      "Type": "string"
    },
    {
      "Name": "street2",
      "Type": "string"
    },
    {
      "Name": "city",
      "Type": "string"
    },
    {
      "Name": "state",
      "Type": "string"
    },
    {
      "Name": "zip",
      "Type": "string"
    },
    {
      "Name": "country",
      "Type": "string"
    },
    {
      "Name": "phone",
      "Type": "string"
    },
    {
      "Name": "fax",
      "Type": "string"
    }
  ],
  "Rows": [
    [
      "S1",
      "Site One",
      null,
      null,
      "Anytown",
      null,
      null,
      null,
      "800-555-1212",
      null
    ]
  ]
}